src/fidesops/ops/core/config.py

```diff
@@ -35,9 +35,10 @@ class ExecutionSettings(FidesSettings):
     """Configuration settings for execution."""
 
     privacy_request_delay_timeout: int = 3600
-    task_retry_count: int
-    task_retry_delay: int  # In seconds
-    task_retry_backoff: int
+    # By default Fidesops will not retry graph nodes
+    task_retry_count: int = 0
+    task_retry_delay: int = 0  # In seconds
```

Comment:
a nit here, but `# In seconds` doesn't help end-users. If we want to clarify it we should update the variable name to contain the unit, i.e. `task_retry_delay_seconds`

Reply:
You don't think all users are going to read the source code first 😄 .

Making this change would be a breaking change, `task_retry_delay` was in previous releases it just gets a default value now. So the question now is it worth it to make a breaking change or could we add something to the docs and that be enough?